src/flows/parser_item_html_text_streaming_flow.py

```diff
@@ -29,6 +29,8 @@
 CHUNK_ROWS = 50000  # 3486 rows = 10MB
 NUM_FILES_PER_RUN = 1000
 
+# location for temp sql file needed to use Prefect Snowflake Task
+TEMP_SQL_FILE_LOC = "/tmp/temp_sql_file.sql"
```

Comment:
Could there be a resource conflict when multiple tasks write to the same file?

Would either of these solutions be easier?
- Use `SnowflakeQuery` to pass the query as a string argument to the task.
- Put the queries in `.sql` files in the project.

Reply:
I like the sql file idea...going to implement that.